### DISCUSSION TITLE
Make M2M relations visible in both related items

### DISCUSSION THREAD

**Discussion by keesvanbemmel** [directus/directus#14167] [Feature Requests]
At the moment, M2M relations are only shown on the current item, not the related item. As shown in this video:

https://user-images.githubusercontent.com/4633876/176313387-935be05d-c6bb-4b65-8be8-08f3981530b5.mov

As discussed [HERE](https://github.com/directus/directus/discussions/14052) it would be a nice feature to be able to show these relations on both ends.

I'd say that on self-referencing relationships this should be the default.

CC @rijkvanzanten

**Comment by rijkvanzanten** [comment]
The main question I have: should that be configurable? Is there a real-life use case where you'd want the self-referencing relationship to _not_ show from both sides? 🤔

**Reply by keesvanbemmel** [reply]
Well I don't think so, but perhaps something like articles in this series that follow this one?

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
